ts/test/tunnels-test/tunnelHostAndClientTests.ts

```diff
@@ -146,11 +146,11 @@ export class TunnelHostAndClientTests {
             clientStreamFactory,
         );
 
-        assert.strictEqual(false, relayClient.isSshSessionActiveProperty);
+        assert.strictEqual(relayClient.isSshSessionActiveProperty, false);
```

Comment:
I fixed the assertion actual/expected parameter order in many places in this file.